scripts/test/Muon/fitting_tab_widget/fitting_tab_presenter_test.py

```diff
@@ -713,6 +714,7 @@ def test_update_fit_status_information_in_view(self):
         self.view.get_index_for_start_end_times = mock.MagicMock(return_value = 1)
         self.view.update_global_fit_state = mock.MagicMock()
         self.view.update_with_fit_outputs = mock.MagicMock()
+        self.presenter.model.start_time_biggest = False
 
```

Comment:
lets add some tests:

fitting_tab_model:  lets check that the start time flag is set in fit_single_Workspace

In the presenter: lets check it swaps the view values correctly